<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeniusHelp</class>
 <widget class="QDialog" name="GeniusHelp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>To enter a word, simply type on your keyboard or select a word from one of the lists. Mark a letter as yellow with a left mouse button click, or as green with a right click. To enter your word, press &quot;Confirm&quot;, or if you made a mistake, click &quot;Reset&quot;.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="buttonOk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset theme="checkbox"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
